### DISCUSSION TITLE
Problem with installing MICOM (Help!)

### DISCUSSION THREAD

**Discussion by SchuShoe** [micom-dev/micom#73] [Q&A] [answered]
Hello, 
I'm having trouble installing the MICOM package on jupyter notebook. 
Could anyone tell me how to install it, I'm new to python and trying to learn.

![Capture](https://user-images.githubusercontent.com/101419498/159066370-adf46c70-b3b8-45ca-9b7d-fb56a9fb7ace.PNG)

Thanks in advance :D

**Comment by cdiener** [comment, marked as answer]
Hi looks like you may be using Python 3.10. MICOM only works with Python 3.6 - 3.10 because none of the solvers that MICOM uses support Python 3.10 yet. If you are using conda or anaconda you can just create a Python 3.9 environment and everything should work.

**Reply by SchuShoe** [reply]
Ok, I see, I'll try with Python 3.9 then 
Thank you for the quick reply :D

**Reply by cdiener** [reply]
Also if you intend to use MICOM more getting a cplex academic license can be worth it. It's much quicker and more accurate than OSQP in MICOM.

**Comment by AlfredoRA** [comment]
Hi! I also have some problems with installing micom. In my case:

Subpakages are not installed by default when I write  :   "pip install micom"   ...   in my cobra environment, so when I use "from micom.data import test_data" (for example) an error says:
" from: can't read /var/mail/micom.data"

I will appreciate any help!
